### PR TITLE
html-forms complete

### PR DESCRIPTION
**Pull request by luiderek**
codingform

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+
```
Comment (Zhiwutian) [inline html-forms/index.html:11]: The big thing missing here is the form tag, make sure you wrap all of the inputs with a form tag

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
form tag needed

File: html-forms/index.html (hunk 1 above)
Comment (luiderek) [reply html-forms/index.html]: Wrapped the form and a few minor tweaks.

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+          <br>
+          Would you like to sign up for more information?
+          <br>
+          <input type="checkbox" name="moreInfo">
+          Sign me up!
+        </label>
+      </div>
+
+      <div>
+        Comments<br>
```
Comment (Zhiwutian) [inline html-forms/index.html:59]: so you want a label for the `comments` input

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
1 final update!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**